local/core/etc/config-settings/unlimited.json

```diff
@@ -24,7 +24,7 @@
                 "fee_read_ledger_entry": 1000,
                 "fee_write_ledger_entry": 3000,
                 "fee_read1_kb": 1000,
-                "bucket_list_target_size_bytes": 2147483648,
+                "bucket_list_target_size_bytes": 2305843009213694000,
```

Comment:
Without this change, quickstart using core version 20.1.0 with unlimited limits wouldn't actually apply unlimited limits do to an exception from the core command line util. The issue is that docker doesn't exit on the failure. I'll look into this.